library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
Alright, so I think I should remove both BootstrapJS and jquery out of headers.inc.php, for now. I'll include them in the files I'll be working on using functions defined here as and when upgrading UI.